pandas/io/sql.py

```diff
@@ -1391,6 +1391,16 @@ class SQLiteTable(SQLTable):
     Instead of a table variable just use the Create Table statement.
     """
 
+    def __init__(self, *args, **kwargs):
+        # GH 8341
+        # register an adapter callable for datetime.time object
+        # if we're here, it means we have sqlite3
+        import sqlite3
+        # this will transform time(12,34,56,789) into '12:34:56.000789'
+        # (this is what sqlalchemy does)
+        sqlite3.register_adapter(time, lambda _: _.strftime("%H:%M:%S.%f"))
+        super(self.__class__, self).__init__(*args, **kwargs)
```

Comment:
Will this work correctly if inherited? You generally want `super(SQLLiteTable, self).__init__(*args, **kwargs)`